backend/src/services/authGoogleService.ts

```diff
@@ -0,0 +1,82 @@
+import { OAuth2Client } from "google-auth-library";
+import { prisma } from "@/database/db";
+import jwt from "jsonwebtoken";
+import { v4 } from "uuid";
+
+const CLIENT_ID =
+  "133300448301-7jr985i9aljbqdbf27okphk58b2f987n.apps.googleusercontent.com";
+const CLIENT_SECRET = "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";
+const REDIRECT_URI = "http://localhost:3008/register/google/callback";
+
+const oAuth2Client = new OAuth2Client(CLIENT_ID, CLIENT_SECRET, REDIRECT_URI);
```

Comment:
it is better to move this `oAuth2Client` to separate file in `backend/src/clients/google` folder.